tree_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	mathrand "math/rand"
 
 	cmn "github.com/tendermint/tendermint/libs/common"
+	"strconv"
```

Comment:
The order of imports should be: 
- stdlib 
- external packages 
- internal packages

Reply:
Good suggestion. I have changed the code.